src/types/config.ts

```diff
@@ -84,7 +84,12 @@ export type EmbedderFrameConfig = {
   origin: string;
 };
 
-export type AnnotationEventType = 'create' | 'update' | 'flag' | 'delete';
+export type AnnotationEventType =
+  | 'create'
+  | 'update'
+  | 'flag'
+  | 'delete'
+  | 'moderate';
```

Comment:
To be honest, I'm not sure if we need this, or where are these activity events consumed. The LMS I guess?

It was easy to add though.

Reply:
Ok, it seems we use this config in the LMS, for Canvas auto submit after an annotation is created or updated.

https://github.com/hypothesis/lms/blob/c9b1a2719d18a8283a82977c2447f717f162dbb7/lms/resources/_js_config/__init__.py#L526-L535

However, the configuration there explicitly defines it only cares about `create` and `update` events, so maybe we can avoid adding yet an extra `moderate` event here, and just add it in future if we have the need.

Reply:
Yes, let's do this. It avoids misleading future readers that the LMS app might be using `moderate` events.